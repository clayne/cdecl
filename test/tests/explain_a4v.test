cdecl @ @ @ explain void a[4] @ 65
