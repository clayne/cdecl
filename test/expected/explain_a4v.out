explain void a[4]
              ^
15: error: array of void; did you mean array of pointer to void?
